src/Umbraco.Web/Editors/LanguageController.cs

```diff
@@ -99,24 +99,28 @@ public Language SaveLanguage(Language language)
                 throw new HttpResponseException(Request.CreateValidationErrorResponse(ModelState));
 
             // this is prone to race conditions but the service will not let us proceed anyways
-            var existing = Services.LocalizationService.GetLanguageByIsoCode(language.IsoCode);
+            var existingByCulture = Services.LocalizationService.GetLanguageByIsoCode(language.IsoCode);
 
             // the localization service might return the generic language even when queried for specific ones (e.g. "da" when queried for "da-DK")
             // - we need to handle that explicitly
-            if (existing?.IsoCode != language.IsoCode)
+            if (existingByCulture?.IsoCode != language.IsoCode)
             {
-                existing = null;
+                existingByCulture = null;
             }
```

Comment:
This is IMO not the controller's responsibility. 

I think we should make an overload/extension method on the service that only returns the language, if the IsoCode matches 100% (Basically executing the code you do here)

Reply:
I didn't change this code, just the var names, I don't exactly know what this is for so I'd prefer to not have to change this in this PR.